p2p/host/routed/routed.go

```diff
@@ -131,7 +130,7 @@ func logRoutingErrDifferentPeers(ctx context.Context, wanted, got peer.ID, err e
 	lm["error"] = err
 	lm["wantedPeer"] = func() interface{} { return wanted.Pretty() }
 	lm["gotPeer"] = func() interface{} { return got.Pretty() }
-	log.Event(ctx, "routingError", lm)
+	// log.Event(ctx, "routingError", lm)
```

Comment:
Why is this code commented?

Reply:
This entire function should be deleted and replaced with a call to `log.Errorw`.

Side note: please (almost) never comment out code unless you're actively debugging.